assets/runtime/functions

```diff
@@ -1643,7 +1643,11 @@ generate_healthcheck_script() {
   ## https://docs.gitlab.com/ee/user/admin_area/monitoring/health_check.html
 cat > /usr/local/sbin/healthcheck <<EOF
 #!/bin/bash
-url=http://localhost${GITLAB_RELATIVE_URL_ROOT}/-/liveness
+if [[ $GITLAB_HTTPS == true ]]; then
```

Comment:
Could you be so kind and use variable expansion by using curly braces, i.e. please replace 

```
if [[ $GITLAB_HTTPS == true ]]; then
```

by

```
if [[ ${GITLAB_HTTPS} == true ]]; then
```

Just to be explicit.

Reply:
DONE. With pleasure